languages/javascript.md

```diff
@@ -432,6 +433,36 @@ else {
 }
 ```
 
+#### Loops
+
+In `for` loops, there must be space between the semi-colons.
+
+`for..in` loops must include a check with `hasOwnProperty`.
+
+We do this:
+
+```js
+for (var i; i < 10; i += 1) {
```

Comment:
"there must be a space _after_ the semi-colons", surely, since the marks themselves are not sequential